devel/libocca/Portfile

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           compilers 1.0
+PortGroup           github 1.0
+PortGroup           legacysupport 1.1
+
+# getline
+legacysupport.newest_darwin_requires_legacy 10
+
+github.setup        libocca occa 1.5.0 v
+name                libocca
+revision            0
+categories          devel parallel
+license             BSD
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         Portable and vendor-neutral framework for parallel programming on heterogeneous platforms
+long_description    OCCA is an open source, portable and vendor-neutral framework for parallel programming \
+                    on heterogeneous platforms. The OCCA API provides unified models for heterogeneous programming \
+                    concepts — such as a device, memory or kernel — while the OCCA Kernel Language (OKL) \
+                    enables the creation of portable device kernels using a directive-based extension to the C language.
+homepage            htpps://libocca.org
```

Comment:
```suggestion
homepage            https://libocca.org
```

Reply:
Thank you for noticing, I need more caffeine, haha.

The port fails at linking atm though, I opened an issue with upstream.

Reply:
Thanks for all your work @barracuda156 👍